locale-source/locale/de/LC_MESSAGES/djangojs.po

```diff
@@ -807,23 +806,22 @@ msgid_plural "%s people have answered."
 msgstr[0] "1 Person hat geantwortet."
 msgstr[1] "%s Menschen haben geantwortet."
 
-#: adhocracy4/polls/static/PollDetail/PollResults.jsx:80
-#: adhocracy4/polls/static/PollDetail/PollResults.jsx:80
-msgid "noone has answered this question"
-msgstr "niemand hat diese Frage beantwortet"
+#: adhocracy4/polls/static/PollDetail/PollResults.jsx:78
+msgid "no one has answered this question"
```

Comment:
Translations are driving me insane again! This is right, @phillimorland did fix that string in a4 and I wanted to add the old trsnalation to then get then on prod and into transifex.

Reply:
Great, I also didn't add the second comment here. The old string was also still there. I will fix the whole commit now, though, so it's not anymore.
So, in short: all is fine now. But without a `make clean` and new install before the `make po`, old a4 strings are still in the djangojs.po files.